Simplenote/Classes/SPAuthViewController.swift

```diff
@@ -65,14 +65,35 @@ class SPAuthViewController: UIViewController {
             passwordWarningLabel.isHidden = true
         }
     }
+    
+    /// # Code: Input Field
+    ///
+    @IBOutlet private var codeInputView: SPTextInputView! {
+        didSet {
+            codeInputView.placeholder = AuthenticationStrings.codePlaceholder
+            codeInputView.passwordRules = UITextInputPasswordRules(descriptor: SimplenoteConstants.passwordRules)
+            codeInputView.returnKeyType = .done
+            codeInputView.textColor = .simplenoteGray80Color
+            codeInputView.delegate = self
+            codeInputView.textContentType = .oneTimeCode
+        }
+    }
+    
+    /// # Code: Warning Label
+    ///
+    @IBOutlet private var codeWarningLabel: SPLabel! {
+        didSet {
+            codeWarningLabel.textInsets = AuthenticationConstants.warningInsets
+            codeWarningLabel.textColor = .simplenoteRed60Color
+            codeWarningLabel.isHidden = true
+        }
+    }
 
     /// # Primary Action: LogIn / SignUp
     ///
     @IBOutlet private var primaryActionButton: SPSquaredButton! {
         didSet {
-            primaryActionButton.setTitle(mode.primaryActionText, for: .normal)
```

Comment:
Most of the UI initialization code that was deleted here, will be handled by the new API(s) `refreshActionViews`  and `reloadInputViewsFromState`.